src/components/flow/ResultsAsSimulation.tsx

```diff
@@ -1,24 +1,366 @@
+		roadFlow: number,
+		point: ExitEntrancePoint,
+	) => {
+		let idx = point.index * -1;
+		let lightColor = LightColors.BLACK;
+		const carNumber = cars.get(point.index)!;
+
+		const result = [];
+
+		for (let i = 0; i < usedLights.length; i++) {
+			const light = usedLights[i];
+			idx = light.index;
+			lightColor = lights.get(usedLights[i].index)!;
+
+			if (i == 0) {
+				result.push(
+					<EEIPointMarker
+						key={idx}
+						color={lightColor}
+						yCord={point.yCord}
+						xCord={point.xCord}
+					>
+						<SimulationNumbers>{carNumber}</SimulationNumbers>
+					</EEIPointMarker>,
+				);
+			} else {
+				result.push(
+					<EEIPointMarker
+						key={idx}
+						color={lightColor}
+						yCord={point.yCord}
+						xCord={point.xCord + 15 * i}
+					></EEIPointMarker>,
+				);
+			}
+		}
+
+		if (result.length == 0) {
+			result.push(
+				// <div key={idx}>
+				<EEIPointMarker
+					key={idx}
+					color={lightColor}
+					yCord={point.yCord}
+					xCord={point.xCord}
+				>
+					<SimulationNumbers>{}</SimulationNumbers>
+				</EEIPointMarker>,
+				// </div>,
+			);
+		}
+		return result;
+	};
 
 	return (
-		<>
+		<ContainerDiv>
 			<Navbar />
-			<div>
-				<h3>Results as simulation</h3>
-				<BaseButtonLink to="../results-choice" relative="path" state={all}>
-					<NeutralNegativeButton>
-						Go back to results choice panel
-					</NeutralNegativeButton>
+			<PageHeader>{crossroadName}</PageHeader>
+			<BorderedWorkaroundDiv>
+				{connections.length > 0 &&
+					connections.map((con) => {
+						const entrancePoint = exitEntrancePoints.filter(
+							(point) => point.id === con.sourceId,
+						)[0];
+						const exitPoint = exitEntrancePoints.filter(
+							(point) => point.id === con.targetId,
+						)[0];
+
+						return (
+							<ConnectionMarker
+								key={con.index}
+								thickness={3}
+								entranceX={entrancePoint.xCord}
+								entranceY={entrancePoint.yCord}
+								exitX={exitPoint.xCord}
+								exitY={exitPoint.yCord}
+								connection={con}
+								color={Colors.PRIMARY_GRAY}
+								withLightIds={true}
+								withTooltip={false}
+							/>
+						);
+					})}
+				{exitEntrancePoints.length > 0 &&
+					exitEntrancePoints.map((point, idx) => {
+						const usedLights = roadsLights.get(point.index)!;
+						return ShowLight(
+							usedLights,
+							lightsSeqCurr,
+							roadFlow.get(point.index)!,
+							point,
+						);
+					})}
+				<CrossroadScreenshot
+					src={crossroadImage}
+					alt="Map screenshot"
+				></CrossroadScreenshot>
+			</BorderedWorkaroundDiv>
+			<tbody>
+				<StyledItemTd>
+					{!running ? (
+						<NeutralNegativeButton
+							onClick={() => setRunning((prev) => !prev)}
+						>
+							Resume
+						</NeutralNegativeButton>
+					) : (
+						<NeutralPositiveButton
+							onClick={() => setRunning((prev) => !prev)}
+						>
+							Pause
+						</NeutralPositiveButton>
+					)}
+				</StyledItemTd>
+			</tbody>
+
+			<NeutralNegativeButton>
```

Comment:
Tutaj trzeba zrobić tą poprawkę dla `<BaseButtonLink />` ale ja to zrobię już po merge'u bo to 2 min roboty.